csrank/choicefunction/fate_choice.py

```diff
@@ -10,6 +10,8 @@
 class FATEChoiceFunction(SkorchChoiceFunction):
     """A variable choice estimator based on the FATE-Approach.
 
+    See the documentation of the ``FATEScoring`` class for more details.
```

Comment:
It could potentially be possible to cross-reference the FATEScoring class:
https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#cross-referencing-syntax

But I am not sure how well that works.